internal/server/v1/optimus/optimus_client.go

```diff
@@ -0,0 +1,22 @@
+package optimus
+
+import (
+	optimusv1beta1grpc "buf.build/gen/go/gotocompany/proton/grpc/go/gotocompany/optimus/core/v1beta1/corev1beta1grpc"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type OptimusClient struct{}
```

Comment:
1. let's restructure the definition by moving this below `OptimusClientBuilder`.
2. This struct naming is misleading, this is not a client but a client builder. We can rename it to e.g. `clientBuilder` or `defaultClientBuilder`.

Reply:
done